### ISSUE TITLE
TypeError in getImageData after adding Affirm Promotional Messaging to our website 

### ISSUE THREAD

**Issue by rafad1v** [GoogleChrome/lighthouse#15625]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.iqair.com/us/air-purifiers/healthpro-series

### What happened?

Encountered an error while using Axe-core on Lighthouse 11.0.0 via https://pagespeed.web.dev/. The issue seems to be specific to pages that include Affirm's Promotional Messaging.

### What did you expect?

For our diagnostic testing to work.

### What have you tried?

I tried contacting Axe Core about this issue, but they directed me back to you guys.

### How were you running Lighthouse?

PageSpeed Insights

### Lighthouse Version

11.0.0

### Relevant log output

```shell
Error Details:

axe-core Error: TypeError: Failed to execute 'getImageData' on 'CanvasRenderingContext2D': Value is not of type 'long'. - feature unsupported in your environment. Skipping color-contrast rule.:
```

**Comment by adamraine** [comment]
#15002. This was fixed in v11.1

**Comment by rafad1v** [comment]
Do you know when 11.1 will be pushed on the actual PageSpeed Insights page?
